<launch>
    <node pkg="tf2_ros" type="static_transform_publisher" name="word_slam" args="0 0 0  0.5, -0.5, 0.5, -0.5  world slam" />
    <node pkg="tf2_ros" type="static_transform_publisher" name="zed_robot" args="0 0 0  0.5, -0.5, 0.5, 0.5  zed robot" />
    <node pkg="disinfslam" type="ros_online_orb3" name="ros_online_orb3"  output="screen" launch-prefix="gdb -ex run --args" >
        <param name="model_path"          value="/home/yu/tools/ht_lt.pt" />
        <param name="calib_path"          value="$(find disinfslam)/configs/zed_28498913_orb3.yaml" />
        <param name="orb_vocab_path"      value="/home/yu/git_ws/ORB_SLAM3/Vocabulary/ORBvoc.txt" />
        <param name="devid"   type="int"       value="10" />
        <param name="renderer"   type="bool"       value="true" />
        <param name="bbox_x_min"   type="double"       value="-8.0" />
        <param name="bbox_x_max"   type="double"       value="8.0" />
        <param name="bbox_y_min"   type="double"       value="-6.0" />
        <param name="bbox_y_max"   type="double"       value="2.0" />
        <param name="bbox_z_min"   type="double"       value="-8.0" />
        <param name="bbox_z_max"   type="double"       value="8.0" />
        <param name="require_mesh"   type="bool"       value="true" />
        <param name="global_mesh"   type="bool"       value="true" />
        <param name="sensor" type="int" value="4" />

    </node>
    <!-- <include file = "$(find TSDF2Mesh)/launch/tsdf2mesh.launch"/> -->
</launch>